lib/sql/delete.js

```diff
@@ -59,13 +59,21 @@ class Delete {
    * @returns {Promise} Returns a promise containing response from server.
    * @example
    * db.delete('posts').all().then(res => ...)
+   * @deprecated
    */
   all() {
     this.params.op = 'all';
     this.options.body = JSON.stringify(this.params);
     let url = sql.sqlURL(this.url, this.db, this.appId, 'crud', `${this.table}/delete`);
     return fetchAsync(url, this.options);
   }
+
+  apply(){
+    this.params.op = 'all';
```

Comment:
@YourTechBud  Is it fine to not give `op` method to sql since it only has one op type?